modules/collabora_online_group/collabora_online_group.install

```diff
@@ -0,0 +1,51 @@
+<?php
+
+use Drupal\views\Entity\View;
+
+/**
+ * Implements hook_install().
+ */
+function collabora_online_group_install(): void {
```

Comment:
Observation: This only works if views is enabled at the time that this module is installed.
Maybe this is ok.
Did you find similar hook_install() in other modules?

Reply:
I've seen examples of changes in configuration on install. 

This case is a bit speciall its modifying configuration from other module on a optional config. If the view was new, with  optional folder would be enough like groupmedia does, but optional configuration doesn't override existing. 

For this case I added changes of the view by code for two reasons:
- Support to groupmedia 3 and 4: group uses different entity types for the relation and that affects to the view, so would mean to handle it somehow by duplicating configuration yaml or something like that. You can see previous commits.
- If we want to stick to groupmedia view we have to stay aligned and  keep ours updated with theirs and fully test the view.